src/net/sourceforge/kolmafia/chat/ChatManager.java

```diff
@@ -236,8 +236,8 @@ public static final StyledChatBuffer getBuffer(final String bufferKey) {
 
     synchronized (ChatManager.bufferEntries) {
       ChatManager.instantMessageBuffers.put(bufferKey, buffer);
-      ChatManager.bufferEntries = new Entry[ChatManager.instantMessageBuffers.size()];
-      ChatManager.instantMessageBuffers.entrySet().toArray(ChatManager.bufferEntries);
```

Comment:
I'm sort of confused about what this line was even doing before, so I'm worried about removing it!

Reply:
same D:

Reply:
hmm ok im willing to go ahead